<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>package_manager_dialog</class>
 <widget class="QDialog" name="package_manager_dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Package manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../eka2l1.qrc">
    <normaloff>:/assets/duck_tank.png</normaloff>:/assets/duck_tank.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="package_list_search_layout">
     <item>
      <widget class="QLabel" name="package_list_search_label">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="package_list_search_edit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="package_list_widget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>UID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vendor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Drives</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="action_package_uninstall">
   <property name="text">
    <string>Uninstall</string>
   </property>
  </action>
  <action name="action_package_list_files">
   <property name="text">
    <string>List files</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../eka2l1.qrc"/>
  <include location="../eka2l1.qrc"/>
 </resources>
 <connections/>
</ui>
